bundles/org.openhab.binding.opensprinkler/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -109,17 +167,51 @@
 		<category>Switch</category>
 	</channel-type>
 
+	<channel-type id="system-channel" system="true" advanced="true">
+		<item-type>Number</item-type>
+		<label>Signal Strength</label>
+		<description>Wi-Fi Signal Strength.</description>
+		<category>QualityOfService</category>
+		<state readOnly="true" pattern="%d %unit%"></state>
+	</channel-type>
```

Comment:
You can simply use any predefined system channel type. You don't need to redefine them.

Reply:
Do I need to change it or can it be left as is? I would prefer to leave it as advanced="true" to encourage people to not link this channel.

Reply:
You can of course leave the channel, but you must not override system channel types.

Reply:
Before removing the definition I could see it by clicking on the equipment group. After removing the definition, it now does not show up as a point in the equipment group despite trying to re-add it with the UI, I can link it and see the value in the item. Have deleted and re-added the thing with the same result.